src/app/controller/services/base/timestamp-definition.service.ts

```diff
@@ -87,6 +86,7 @@ export class TimestampDefinitionService {
 
   getTimestampDefinitionsMap(): Observable<Map<string, TimestampDefinitionTO>> {
     return this.getTimestampDefinitions().pipe(
+      take(1),
```

Comment:
I do not think we need the `take(1)` here as we are returning the observable? (Maybe we need a `this.defintionMapCache$` attribute to store the observable in and have it be a `shareReplay(1)` as well).

Reply:
I agree as we repeatedly do this mapping, which is a waste. I will also replay the `getNegotiationCycles` observable output to be consistent. NegotiationCycles can also make use of the Async Pipe. That I will also do as part of this.